kubernetes/linux/main.sh

```diff
@@ -394,15 +394,15 @@ fi
 
 #Parse the configmap to set the right environment variables for MDM metrics configuration for Alerting.
 if [ "${CONTAINER_TYPE}" != "PrometheusSidecar" ]; then
-      /usr/bin/ruby2.6 tomlparser-mdm-metrics-config.rb
+      /usr/bin/ruby2.7 tomlparser-mdm-metrics-config.rb
 
       cat config_mdm_metrics_env_var | while read line; do
             echo $line >> ~/.bashrc
       done
       source config_mdm_metrics_env_var
 
       #Parse the configmap to set the right environment variables for metric collection settings
-      /usr/bin/ruby2.6 tomlparser-metric-collection-config.rb
+      /usr/bin/ruby2.7 tomlparser-metric-collection-config.rb
```

Comment:
since we are updating ruby version, along with functional validation, please verify the fluentd.log file doesnt have any exceptions due to ruby version upgrade,

Reply:
> since we are updating ruby version, along with functional validation, please verify the fluentd.log file doesnt have any exceptions due to ruby version upgrade,

since you have validated on ruby2.7 so resolving this comment.